config/epic.yml

```diff
@@ -13,6 +13,7 @@ auth:
   sign_in_steps:
     - element: "#Login"
       send_keys: "fhirjason"
+      wait: 5
```

Comment:
Can you help me understand this wait? Normally I thought we wait (until a default timeout) for every element to appear before proceeding.